_sass/_layout.scss

```diff
@@ -295,6 +299,71 @@
         .hidden {
             display: none;
         }
+
+        .impact-totals {
+            .dashboard {
+                display: -webkit-box;
```

Comment:
can we add a mixin to handle vendor prefixes? or maybe compass for out of the box stuff